src/modules/previewpane/common/Telemetry/TelemetryBase.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Eventing.Reader;
+using System.Diagnostics.Tracing;
+
+namespace PreviewHandlerCommon.Telemetry
+{
+    /// <summary>
+    /// Base class for telemetry events.
+    /// </summary>
+    public class TelemetryBase : EventSource
+    {
+        /// <summary>
+        /// The event tag for this event source.
+        /// </summary>
+        public const EventTags ProjectTelemetryTagProductAndServicePerformance = (EventTags)0x0u;
+
+        /// <summary>
+        /// The event keyword for this event source.
+        /// </summary>
+        public const EventKeywords PROJECTKEYWORDMEASURE = (EventKeywords)0x0;

```

Comment:
this variable naming convention is inconsistent with the rest in the class. For all caps variable naming convention I suggest we add "_" underscore for more readability. For example, PROJECTKEYWORDMEASURE can be written as PROJECT_KEYWORD_MEASURE. https://stackoverflow.com/questions/838929/naming-why-should-named-constants-be-all-uppercase-in-c-java

Reply:
This is not allowed by stylecop.

Reply:
Sorry, I thought I added this comment yesterday.  Generally for C# we should be using the [dotnet naming guidlines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines ).  consts are usually Pascal Case.

Reply:
Sure Ryan! I will update this.